### PR TITLE
Added Xcode in Dark Mode

### PR DESCRIPTION
**Pull request by jadebowl**

### CODE REVIEW

File: Example/Highlightr.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -603,6 +485,7 @@
 			baseConfigurationReference = 4CF75D031FDF606D266D5CFE /* Pods-Highlightr_Example.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = RV92JH4QP5;
```
Comment (raspu) [inline Example/Highlightr.xcodeproj/project.pbxproj:488]: Please remove your development team

File: Example/Highlightr.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -619,6 +502,7 @@
 			baseConfigurationReference = FACBE1ED06E5718AE417DCC0 /* Pods-Highlightr_Example.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = RV92JH4QP5;
```
Comment (raspu) [inline Example/Highlightr.xcodeproj/project.pbxproj:505]: Please remove your development team

**Review by raspu — COMMENTED** [review]
Thanks a lot! It looks 👌, just a small thing.

File: Example/Highlightr.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jadebowl) [reply Example/Highlightr.xcodeproj/project.pbxproj]: Hi! I thought I had removed it but forgot to do so in the Example Project. Should be fixed now.

03dd0f36334ee12d8215e52290b1fc617d255b80

File: Example/Highlightr.xcodeproj/project.pbxproj (hunk 2 above)
Comment (jadebowl) [reply Example/Highlightr.xcodeproj/project.pbxproj]: Hi! I thought I had removed it but forgot to do so in the Example Project. Should be fixed now.

03dd0f36334ee12d8215e52290b1fc617d255b80

**Pull request merged**